File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Soloists/SoloistResultSoloistDto.ts
import { SoloistResultArtistDto } from "./SoloistResultArtistDto";
import { SoloistResultConcertDto } from "./SoloistResultConcertDto";
import { SoloistResultGenderDto } from "./SoloistResultGenderDto";
import { SoloistResultRoleArtistDto } from "./SoloistResultRoleArtistDto";

export class SoloistResultSoloistDto {
    musicalProtagonistId: Number = 0;
    name: String = "";
    startDate: Date = new Date();
    gender: SoloistResultGenderDto = new SoloistResultGenderDto();
    artist: SoloistResultArtistDto = new SoloistResultArtistDto();
    concerts: Array<SoloistResultConcertDto> = new Array();
    roleArtist: SoloistResultRoleArtistDto = new SoloistResultRoleArtistDto();
}

File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Soloist/SoloistResultSoloistDto.cs
using System;
using System.Collections.Generic;

namespace ArenaGestor.APIContracts.Soloist
{
    public class SoloistResultSoloistDto
    {
        public int MusicalProtagonistId { get; set; }

        public string Name { get; set; }

        public DateTime StartDate { get; set; }

        public SoloistResultGenderDto Gender { get; set; }

        public SoloistResultArtistDto Artist { get; set; }

        public List<SoloistResultConcertDto> Concerts { get; set; }

        public SoloistResultRoleArtistDto RoleArtist { get; set; }

    }
}
